### DISCUSSION TITLE
PCB price

### DISCUSSION THREAD

**Discussion by vitasam** [markadev/AppleII-VGA#45] [General]
Hi,
the PCB size is 75mm x 113mm. There is a significant price jump on JLCPCB if the board is above 100mm on a single dimension. 
Will it make sense to squeeze the board a little bit below 100mm?

**Comment by markadev** [comment]
You can try it but it may be a challenge due to the way the chips are oriented.

**Reply by vitasam** [reply]
I'm trying it to do for the MPF-II version of the VGA card, but I don't know is it possible to resize the PCB edge cuts (at least I was not able so far)

**Reply by vitasam** [reply]
Solved here: https://github.com/markadev/AppleII-VGA/discussions/47
